src/WinRPM.jl

```diff
@@ -459,25 +465,19 @@ function do_install(package::Package)
         error("failed to download $name $(data[2]) from $source/$path.")
     end
     cache = getcachedir(source)
-    path2 = joinpath(cache,escape(path))
+    path2 = joinpath(cache, escape(path))
     open(path2, "w") do f
         write(f, data[1])
     end
     info("Extracting: ", name)
-
-    if VERSION < v"0.7.0-DEV.2181"
-        cpio = splitext(path2)[1]*".cpio"
-    else
-        cpio = splitext(joinpath(cache, escape(basename(path))))[1] * ".cpio"
-    end
-
+    cpio = splitext(path2)[1]*".cpio"
```

Comment:
wrong way around, should use the newer code

Reply:
but that was actually failing for me!